mpt_trie/src/debug_tools/diff.rs

```diff
@@ -41,7 +41,7 @@ use crate::{
 /// [branch][`Node::Branch`]s have no [`Nibble`] directly associated with them.
 fn get_key_piece_from_node<T: PartialTrie>(n: &Node<T>) -> Nibbles {
     match n {
-        Node::Empty | Node::Hash(_) | Node::Branch { .. } => Nibbles::default(),
```

Comment:
nit: why removing all calls to `Nibbles::default()` especially after providing a custom impl of `Default`?

Reply:
So my thought was it was a bit more easier to read. Like what's important to know here is that we are creating a `Nibbles` that is empty, which might not be as clear from calling `default`.

This is really a minor nit though, so I'm cool either way.

Reply:
Hmm yeah if we end up switching to `new`, then I would say lets just stick with `default`, as they are pretty much the same.

Reply:
Yeah it was more like a general question, I'm good either way.